theatre/shared/src/notify.ts

```diff
@@ -0,0 +1,57 @@
+import logger from './logger'
```

Comment:
Aria and I discussed deprecating / removing usages of this logger previously (because of the implementation is complex and most usages of it require a logger instance which is hard to maintain). Could be replaced with a console.log?

Reply:
I'd say let's treat the deprecation of the logger in a future PR, this one should be focused on notifications. Usages should be easy to find and replace, so I'd keep things consistent for now and keep using logger here.

Reply:
Makes sense to me